datacontract/export/avro_converter.py

```diff
@@ -40,7 +40,10 @@ def to_avro_field(field, field_name):
     avro_field = {"name": field_name}
     if field.description is not None:
         avro_field["doc"] = field.description
-    avro_field["type"] = to_avro_type(field, field_name)
+    if field.required is None:
+        field.required = True
```

Comment:
I don't get this. Why do you set field.required to True if the property is set (cloud also be false, couldn't it)?

Reply:
This part of the code was to handle cases where the yaml field did not have a required attribute set (so neither true or false). My initial thought was to set that field to true in those cases since all fields in avro are required (unless specified otherwise). I was able to remove this part and still handle those cases correctly. The test I added shows the three different cases